core/src/main/scala/kafka/tools/DumpLogSegments.scala

```diff
@@ -38,6 +38,7 @@ object DumpLogSegments {
     val parser = new OptionParser
     val printOpt = parser.accepts("print-data-log", "if set, printing the messages content when dumping data logs")
     val verifyOpt = parser.accepts("verify-index-only", "if set, just verify the index log without printing its content")
+    val indexSanityOpt = parser.accepts("verify-index-sanity", "if set, just verify the index sanity without printing its content")
```

Comment:
@ijuma changed the option name and updated the description to make it clear where this check is used.